core-bundle/src/Controller/SitemapController.php

```diff
@@ -76,6 +76,7 @@ public function __invoke(Request $request): Response
         $urlSet = $sitemap->createElementNS('https://www.sitemaps.org/schemas/sitemap/0.9', 'urlset');
 
         foreach ($urls as $url) {
+            $url = htmlspecialchars($url, ENT_XML1, $GLOBALS['TL_CONFIG']['characterSet'] ?? 'UTF-8', false);
```

Comment:
`Config::get('characterSet')` is deprecated. In a proper service/controller you can just use `%kernel.charset%` (which `Config::get('chracterSet')` gets its value from).

Reply:
changed in 72332ea506cc5ee13ba8f66c13981a28af44cf66